File: ex2.py
while True:
    try:
        n=int(input("Please enter a number: "))
        if n>0 :
            break
    except ValueError:
        print("Wrong input!")

odd = 1
num_record=[1]

for _ in range(n):
    odd+=2
    print(" ".join(map(str,num_record)))
    num_record.append(odd)

File: ex2.js
let input = prompt("Please enter a positive number: ")
if (!isNaN(input) && input>0 && input.trim !==" "){
    let odd=1
    let number_rec = []
    for (let i = 0; i < input; i++){
        number_rec.push(odd)
        console.log(number_rec.join(" "));
        odd +=2
        
    }
}
else{
    console.log("Invalid input")
}